package.json

```diff
@@ -48,7 +48,7 @@
   "main": "webpack.config.js",
   "keywords": [],
   "dependencies": {
-    "identity-style-guide": "^0.2.4",
+    "identity-style-guide": "18f/identity-style-guide#bh-add-app-styles",
```

Comment:
@hursey013 Could we just wait until the new version of `identity-style-guide` is updated and then set that version here?